easybuild/easyconfigs/p/PyTorch/PyTorch-0.3.1-foss-2018a-Python-3.6.4-CUDA-9.1.85.eb

```diff
@@ -0,0 +1,97 @@
+easyblock = 'PythonPackage'
+
+name = 'PyTorch'
+version = '0.3.1'
+cuda_ver = '9.1.85'
+versionsuffix = '-Python-%%(pyver)s-CUDA-%s' % cuda_ver
+
+homepage = 'http://pytorch.org/'
+description = """Tensors and Dynamic neural networks in Python with strong GPU acceleration.
+PyTorch is a deep learning framework that puts Python first."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+# This environment is needed to prevent setup.py from invoking git to determine version.
+prebuildopts = 'PYTORCH_BUILD_VERSION=%(version)s PYTORCH_BUILD_NUMBER=1'
+
+# you can choice here: either give a list of CUDA cc version or tell it All
+# by default it does autodetect of the GPU on the local machine
+prebuildopts += ' TORCH_CUDA_ARCH_LIST="3.0 3.2 3.5 3.7 5.0 5.2 5.3 6.0 6.1 7.0"'
+
+pytorchdir = 'pytorch-%(version)s'
+extract_cmd_pattern = 'tar -C %s/%s --strip-components=1 -xf %%s'
+
+source_urls = [
+    'https://github.com/pytorch/pytorch/archive',
+    "https://github.com/facebookincubator/gloo/archive",
+    "https://github.com/google/googletest/archive",
+    "https://github.com/nanopb/nanopb/archive",
+    "https://github.com/pybind/pybind11/archive",
+    "https://github.com/wjakob/clang-cindex-python3/archive",
+]
+sources = [
+    'v%(version)s.tar.gz',  # PyTorch
+    {
+        'filename': 'cb002e4eb8d167c2c60fc3bdaae4e1844e0f9353.tar.gz',
```

Comment:
@wpoely86 Can we use `download_file` here, and rename the downloaded file using `filename` to a more meaningful filename like `gloo-<datestamp-of-commit>.tar.gz` (same below)?